kclvm/sema/src/resolver/arg.rs

```diff
@@ -7,21 +7,39 @@ use std::rc::Rc;
 use crate::resolver::pos::GetPos;
 
 impl<'ctx> Resolver<'ctx> {
+    fn get_func_name(&mut self, func: &ast::Expr) -> String {
+        let mut callee = func;
+        loop {
```

Comment:
why loop here?

Reply:
We need to extract the function pointer inside the ParenExpr,for example:
```
f = lambda x {x}
a = (((f)))(1)
```